manifests/get_from_github.pp

```diff
@@ -1,17 +1,26 @@
-# private
-define collectd::get_from_github(
-  $localfolder,
-  $source
+define collectd::get_from_github (
+  $localfolder = undef,
+  $source      = undef,
+  $ensure      = present,
+  $revision    = 'master',
 ) {
-  if(!defined(Vcsrepo[$localfolder])){
-    collectd::check_and_install_package { $title:
-      package_name => 'git'
-    } ->
+  validate_string($source)
+  validate_string($revision)
```

Comment:
metadata.json has stdlib as a dependency, also the validate_hash function is used in every plugin/*.pp so this is not a new dependency being added just for this class. 

This class is only used by rabbitmq currently and seems like a one off, since a majority of other plugins are included in this puppet module I do not see why rabbitmq would be treated differently. Not sure if anyone is using this or now, but we could remove this class as it would also remove the git and vcsrepo dependencies from the module